src/app/components/competitions/competition-card/competition-card.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Competition } from 'src/app/entity/competitions.model';
+import { Status } from 'src/app/entity/Status';
+import { StatusIcons } from 'src/app/entity/Status';
+
+@Component({
+  selector: 'app-competition-card',
+  templateUrl: './competition-card.component.html',
+  styleUrls: [
+    './competition-card.component.scss',
+    '../../../shared/styles/styled-card.scss',
+  ],
+})
+export class CompetitionCardComponent implements OnInit {
+  @Input() competition: Competition = {
+    id: 0,
+    name: '',
+    description: '',
+    creator: '',
+    users: [],
+    userGroups: [],
+    userProfiles: [],
+    events: [],
+    created: new Date(),
+    lastModified: new Date(),
+    status: Status.DRAFT,
+  };
+
+  @Output() deleteEmitter = new EventEmitter<void>();
+  @Output() viewEmitter = new EventEmitter<void>();
+  @Output() editEmitter = new EventEmitter<void>();
+
+  statusIcon = StatusIcons[this.competition.status];
+
+  ngOnInit(): void {
+    this.statusIcon = StatusIcons[this.competition.status];
+  }
+
+  emitViewClick() {
```

Comment:
I would get rid of these methods and just emit the events directly from the HTML template